web/packages/teleport/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "start": "yarn build-wasm && vite",
-    "build-wasm": "cargo install --locked wasm-bindgen-cli && wasm-pack build ./src/ironrdp --target web --mode no-install",
+    "build-wasm": "cargo install --locked --force wasm-bindgen-cli --version $(cargo metadata --format-version 1 | jq -r '.packages[] | select(.name == \"wasm-bindgen\") | .version') && wasm-pack build ./src/ironrdp --target web --mode no-install",
```

Comment:
At this point, we're basically re-implementing what wasm-pack does automatically.

I' tempted to get rid of the cargo install for wasm-bindgen-cli and remove the `--mode no-install` flag.